core/src/gl/shaderProgram.cpp

```diff
@@ -292,7 +292,14 @@ std::string ShaderProgram::applySourceBlocks(const std::string& source, bool fra
     //     }
     // }
 
-    return sourceOut.str();
+    // Certain graphics drivers have issues with shaders having line continuation backslashes "\".
+    // Example raster.glsl was having issues on s6 and note2 because of the "\"s in the glsl file.
+    // This also makes sure if any "\"s are present in the shaders coming from style sheet will be
+    // taken care of.
+    auto str = sourceOut.str();
+    std::regex backslashMatch("\\\\ *?\\n");
```

Comment:
I'm confused about the ` *?` part of your regex -- removing spaces is usually done by matching `\s`, and the `?` should be unnecessary after the `*`. Shouldn't `\\s*` (double-escaped backslash) work?